content/ux-guide/research/accessibility.md

```diff
@@ -0,0 +1,120 @@
+
+- **Focus on connecting at a human level.** Validate the participant’s experiences, especially when they’ve shared something difficult about themselves or their disability. Help them feel comfortable and don’t rush into usability questions when a participant opens up about painful situations.
+
+- **Offer multiple ways to respond.** Let the participant choose how they want to accomplish a task or respond, such as verbally, in chat, by phone, or cameras off.
+
+- **Speak up as needed.** It’s okay to step in or answer questions to reassure participants. Long silences don’t work for everyone. While we often leave space for participants to navigate and think aloud, some participants may need more verbal feedback. Words of encouragement like “yes” and “great callout” can build rapport. If you’re concerned about leading the participant to an answer, ask them what they expect or need to be successful. Keep in mind that visual cues like nodding or smiling don’t always translate well.
+
+### Pay attention to time
+
+Build unstructured time into your interview guide to account for differences in processing and response time. Make sure the script allows for detours and cutting later tasks if needed.
+
+- **Be flexible with scheduling.** Some participants may prefer morning or afternoon sessions, depending on their needs, energy levels, and access to transportation (when meeting in person or using a computer outside the home).
+
+- **Don’t overbook the session.** Prioritize key questions and allow yourself time to ask follow-up questions about a participant’s needs. Some participants may not complete all of the testing, whether it’s because of a disability, a more talkative style, Wi-Fi trouble, or some other reason.
+
+- **Offer breaks and time checks.** Participants may fatigue and need shorter sessions or breaks between tasks. In addition, time checks can help participants stay focused and have a sense of time passing.
+
+- **Practice the session timing with a colleague or agency partner.** Invite interested peers to do a dry run if they’re comfortable talking about their disability or cognitive needs to improve the research protocol. Within TTS, you could try the #neurodiversity or #g-accessibility Slack channels to find people.
+
+### After the session
+
+- **Focus on user needs over limitations.** If someone is having difficulty navigating a website, it may have nothing to do with their disability or tech proficiency. When synthesizing things in archetypes or user stories, for example, tie insights to the user’s needs or goals rather than a particular disability.
+
+- **Keep talking about accessibility.**
```

Comment:
The primary audience for the UX guide has been redefined as the public. With this guide as an open source resource that others can adapt and reuse. More is still to be hashed out in content strategy. I'd remove this section for now

Reply:
Added a commit for this